packages/botonic-cli/src/util/bot-config.ts

```diff
@@ -53,6 +53,7 @@ export class BotConfig {
       botonicCli.match(botonicCliVersionRegex)?.[1] || ''
 
     const configLoaded = await this.loadBotConfig(appDirectory)
+    fs.rm(path.join(appDirectory, 'dist', 'bot-config.js'))
```

Comment:
Doubt: What happens if `@botonic/cli` is updated to the latest version and the `rspack.config.js` file is not updated with the needed changes? Will this removal fail and exit the deploy process?

Reply:
I have checked it, if this file does not exist it gives an error and the deploy is not done. I refactor the code to remove the file inside the try catch. This way if it fails we will see the log but the deploy will work the same.